### DISCUSSION TITLE
是不是可以加个实验选项，把CSS渐变封面应用的更广

### DISCUSSION THREAD

**Discussion by b1xcy** [theme-shoka-x/hexo-theme-shokaX#348] [Ideas]
就是把上\下一篇的的封面也改为CSS渐变
我这边自己改好了，有需要我直接发个pr
![image](https://github.com/user-attachments/assets/36c9bc8a-b0ff-49da-a6d7-1d76f17bc302)

**Comment by zkz098** [comment]
发PR就行
配置可以都放实验性里面，后期会安排配置转正
